src/content/whats-new/2024/10/whats-new-10-10-pathpoint.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Connect telemetry to business KPIs with New Relic Pathpoint'
+summary: 'Get a strategic view of how your system performance impacts real-world business outcomes'
+releaseDate: '2024-10-10'
+learnMoreLink: 'https://docs.newrelic.com/docs/new-relic-solutions/business-observability/install-pathpoint/'
+getStartedLink: 'https://onenr.io/0oQDP60y7Qy'
+---
+
+New Relic has released a new version of Pathpoint, built on an entirely new code base featuring significant enhancements and AI capabilities. First introduced in January 2024, this version offers deeper business insights with a unique journey flow layout that visualizes stages, steps, and signals to create a shared understanding of business health for both frontline teams like customer support and higher-level stakeholders such as product managers, business analysts, and C-level executives.
```

Comment:
```suggestion
New Relic has released a new version of Pathpoint, built on an entirely new code base that features significant enhancements and AI capabilities. First introduced in January 2024, this version offers deeper business insights with a unique journey flow layout that visualizes stages, steps, and signals to create a shared understanding of business health for both frontline teams like customer support and higher-level stakeholders such as product managers, business analysts, and C-level executives.
```

Reply:
The original copy was a good start but it was a bit abstract and vague. I've made some changes and broken up the complex ideas to its more parseable. Let me know if you want to change anything about the updated copy.